backend/src/database/migrations/init.py

```diff
@@ -0,0 +1,43 @@
+from src.database.models import (
+    DbAlbum,
+    AlbumArtistRelationship,
+    AlbumGenreRelationship,
+    DbArtist,
+    DbGenre,
+    DbPlaylist,
+    PlaylistAlbumRelationship,
+    DbUser,
+    database,
+)
+
+
+def up():
```

Comment:
I can't see where/how this is called, and so my attempts to run the app locally failed; even if it's a bit hacky for now, probably worth adding some basic instructions for this to the Readme!